cli/flox-rust-sdk/src/providers/publish.rs

```diff
@@ -763,16 +767,33 @@ fn check_build_metadata_from_build_result(
             .collect(),
     );
 
-    let outputs_to_install: Vec<String> = build_result.outputs.clone().into_keys().collect();
+    // Get outputs to install from the build result, or default to all outputs.
+    let outputs_to_install = build_result
+        .meta
+        .outputs_to_install
+        .clone()
+        .unwrap_or_else(|| outputs.0.iter().map(|o| o.name.clone()).collect());
```

Comment:
Do we intend to define installed output at _scrape time_?
I'm asking becasue this is likely exhibiting different semantics than nixpkgs, unless the catalog is _also_ defaulting to all outputs there.
It certainly differs from nix where iirc nix will default to _the first_ defined output. 

https://nix.dev/manual/nix/2.25/language/derivations#:~:text=You%20can%20refer%20to%20each,myPackage%20is%20equivalent%20to%20myPackage.lib.